weblate/templates/snippets/project/billing-libre-trial.html

```diff
@@ -6,8 +6,10 @@
 {% block message %}
   {% comment %}user_can_view_billing is defined in snippets/project/state.html{% endcomment %}
   {% if user_can_view_billing %}
-    <a href="{{ object.billing.get_absolute_url }}" class="btn btn-primary pull-right flip">{% trans "Request approval for libre hosting" %}</a>
+    <a href="{{ object.billing.get_absolute_url }}" class="btn btn-primary pull-right flip">{% trans "Request approval for Libre hosting" %}</a>
   {% endif %}
-  {% trans "This project is in the trial period, so be cautious before you contribute. Things like license or repository can still change before being approved as a libre project." %}
+  {% trans "Hey translator! This project is in a trial period, and its setup might not be complete yet!" %}
+  <br/>
+  {% trans "Consider asking its maintainers before you contribute to avoid the chance of losing your work. Remember that before approval as a libre project, essential things like a license can change." %}
```

Comment:
"its" is now split from the rest, and so is "(project)".
You thereby get misunderstandings and misgendered nouns.
"a license" is not quoted, and not preceded by a comma, and would be bad language if it did.

Maintainers don't want to be asked about people contributing at all. Make it more apparent that the info and announcements and locking should be used if someone people shouldn't contribute.

Reply:
> "its" is now split from the rest, and so is "(project)".

Are you referring to the first or the second line?

> Maintainers don't want to be asked about people contributing at all.

I would agree.

> Make it more apparent that the info and announcements and locking should be used if someone people shouldn't contribute.

I disagree that locking should be used (see my reply in the discussion thread). More importantly: this messaging is targeting potential translators, so including instructions to project maintainers doesn't make much sense IMHO.

Reply:
It’s split into two messages now, translators will see one, the other is for admins. 

Feedback is incorporated, further is welcome.